src-java/kilda-model/src/main/java/org/openkilda/model/BfdProperties.java

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.Duration;
+
+@Getter
+@ToString
+@EqualsAndHashCode
+public class BfdProperties {
+    @JsonProperty("interval")
+    protected final Duration interval;
```

Comment:
I wouldn't use Duration here, it force to create new db converter(converters in our case not the most reliable thing ever),
mappers, etc. While the actual usage of it as duration is made in single place only

Reply:
This is the easiest and simple way to store time duration in an unambiguous way.

Regarding issues with converters - first point - we are going to move out neo4j soon enough, the second we do not have a clear vision where the issue and is there any relation to the number of converters. So I don't see the reason to not use proper/handy data types in our model.

Regarding mappers - all other codebase uses `Duration` for this field, so if we use a different type here we must use mappers too.